umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -23,6 +23,37 @@
   (let [value (subs str start (some #(when % %) args))]
     (str/trim (subs value (inc (.indexOf value ":"))))))
 
+(def description-string-field-re-pattern
+  "Returns the pattern that matches all the related fields in description-string"
+  (let [pstr (str "URLContentType:|Description:|Type:|Subtype:")]
+    (re-pattern pstr)))
+
+(defn- convert-description-string-to-map
+  "Convert Description string to a map, removing fields that are empty or nil.
+  Description string: \"URLContentType: DistributionURL, Description: A very nice URL, Type: GET DATA, Subtype: Subscribe\"
```

Comment:
I removed the commas because they are not supposed to be there. If they are there then the will be part of the text when translated and some of the values will not be valid (DistributionURL, Type, and Subtype).